### PR TITLE
Added in moon option for plot_airmass

### PR DESCRIPTION
**Pull request by cagostino**
I think this would be a useful feature to have in plot_airmass as it's easily calculable just with the information given and I think this would be a common enough calculation people have to do that it would be worth having to save a little bit of time.

### CODE REVIEW

File: astroplan/plots/time_dependent.py [hunk 1]
```diff
@@ -120,7 +118,8 @@ def plot_airmass(targets, observer, time, ax=None, style_kwargs=None,
     max_region : float
         If set, defines an interval between ``max_airmass`` and ``max_region``
         that will be shaded. Default is `None`.
-
+    moon_airmass : bool
```
Comment (wtgee) [inline astroplan/plots/time_dependent.py]: @cagostino it looks like you just forgot to include the `moon_airmass` parameter in the method declaration (lines 47-48).

I'd be curious about this PR. Can you make the fix above and post an example plot of what this looks like? Cheers!

Comment (cagostino) [reply astroplan/plots/time_dependent.py]: 3xmmhii0airmass
I've added an example. Here red is the corresponding airmass for the moon. I should be able to make these suggested changes as well as soon as my semester ends.

If you'll notice, the brightness shading here also covers multiple days, which is the subject of a separate pull request (#354) and is not currently implemented in the master branch iirc.

**Comment by wtgee** [comment]
@cagostino any way you can clean this up and get submitted? If you don't have time let me know. I'm at a hack-week this week and might try to clean up some of the airmass plots so would just incorporate this. Same with #354

**Comment by cagostino** [comment]
@wtgee yeah probably. I was not exactly sure what to do with #354 since its checks were no longer working and weren't showing any details as to why. Would be happy to get that one through though.

**Comment by cagostino** [comment]
I fixed what you had suggested previously, let me know if there is anything else I can do to move this along @wtgee

**Comment by wtgee** [comment]
> I fixed what you had suggested previously, let me know if there is anything else I can do to move this along @wtgee

Thanks! There's a conflict in one of the files still, if you could clean that up @bmorris3 can then take a look for merging.

**Pull request closed without merging**